cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,18 +1,22 @@
 apiVersion: v1beta3
 kind: ReplicationController
 metadata: 
+  name: monitoring-heapster-v1
```

Comment:
meta-comment: if you're going to clean all these up, can you drop a README .. maybe in in cluster/addons ..  that describes the current contract of what's needed/wanted in an add-on?